src/stravalib/client.py

```diff
@@ -1262,6 +1260,10 @@ def get_segment_effort(self, effort_id: int) -> model.SegmentEffort:
         class:`stravalib.model.SegmentEffort`
             The specified effort on a segment.
 
+        Notes
+        ------
+        https://developers.strava.com/docs/reference/#api-SegmentEfforts
+
         """
         return model.SegmentEffort.model_validate(
             self.protocol.get("/segment_efforts/{id}", id=effort_id)
```

Comment:
This endpoint [still exists in the docs as far as I can see](https://developers.strava.com/docs/reference/#api-SegmentEfforts-getSegmentEffortById).  But it could use more testing (I think?)

Reply:
This is the endpoint we _should_ be using, so nothing needs to change here.

Reply:
Perfect!! now that I have a better grasp of our test mock setup and mocking in general, I may try to fill in some test gaps (but in a separate PR). Let's get this one merged!